chimney/src/test/scala/io/scalaland/chimney/TotalTransformerMutableFieldSpec.scala

```diff
@@ -0,0 +1,100 @@
+package io.scalaland.chimney
+
+import scala.annotation.unused
+
+class TotalTransformerMutableFieldSpec extends ChimneySpec {
```

Comment:
This class is basically just a copy of `TotalTransformerDslsSeparationSpec` but with `var` fields on all Foo/Bar case classes. There may be a better place for the tests to go, but I'm not sufficiently familiar with the structure of the project... Nonetheless, these tests do pass without the accompanying fix

Reply:
I'd say that for this issue adding a new entry in `IssuesSpec` would be enough. Just use enough transformations to cover all edge cases (transformInto/into.transform + maybe ditto for partials). If you want you can use either one test (`test("fix issue #400")`) or separate tests but grouped (`group("fix issue #400") { /* tests */ }`).

Reply:
Ah great that sounds much neater, I'll rework the tests a little today then. Thanks for your quick response!

Reply:
Pushed neater tests 👍